src/MainLayout.tsx

```diff
@@ -21,6 +23,21 @@ const LoadableStudioLayout = loadable(() => import('./views/studio/StudioLayout'
 })
 
 const MainLayout: React.FC = () => {
+  const [openDialog, closeDialog] = useDialog({
+    title: 'Outdated browser detected',
+    description: 'It seems your browser is outdated, the Joystream app may not work properly',
```

Comment:
To be in line with similar dialog we already have:

```suggestion
    description: 'It seems the browser version you are using is not fully supported by Joystream. Some of the features may be broken or not accessible. For the best experience, please upgrade your browser to the latest version.
```

Reply:
I think it would also be nice to include a link with some instructions. For example https://www.whatismybrowser.com/guides/how-to-update-your-browser/auto will automatically show you instructions for your browser. That could be `Click here to see instructions.` at the end. Wdyt?

Reply:
Second this